functional-test/src/test/groovy/org/rundeck/tests/functional/api/jobs/JobScheduledSpec.groovy

```diff
@@ -0,0 +1,214 @@
+                response.retriedExecution.id != null
+            }
+        when:
+            def responseExec1 = JobUtils.waitForExecutionToBe(
+                    ExecutionStatus.TIMEDOUT.state,
+                    response.retriedExecution.id as String,
+                    new ObjectMapper(),
+                    client,
+                    1,
+                    WaitingTime.MODERATE.milliSeconds
+            )
+        then:
+            verifyAll {
+                responseExec1.status == 'timedout'
+                response.retriedExecution != null
+            }
+    }
+
+    def generatePath() {
+        def xmlJob = "<joblist>\n" +
+                "   <job>\n" +
+                "      <name>scheduled job</name>\n" +
+                "      <group>api-test/job-run-scheduled</group>\n" +
+                "      <uuid>api-test-job-run-scheduled</uuid>\n" +
+                "      <description></description>\n" +
+                "      <loglevel>INFO</loglevel>\n" +
+                "      <context>\n" +
+                "          <project>xml-project-name</project>\n" +
+                "      </context>\n" +
+                "      <dispatch>\n" +
+                "        <threadcount>1</threadcount>\n" +
+                "        <keepgoing>true</keepgoing>\n" +
+                "      </dispatch>\n" +
+                "      <schedule>\n" +
+                "        <time hour='P_NH' seconds='P_NS' minute='P_NM' />\n" +
+                "        <month month='P_NMO'  day='P_ND' />\n" +
+                "        <year year='P_NY' />\n" +
+                "      </schedule>\n" +
+                "      <timeZone>GMT</timeZone>\n" +
+                "\n" +
+                "      <sequence>\n" +
+                "        <command>\n" +
+                "        <exec>xml-args</exec>\n" +
+                "        </command>\n" +
+                "      </sequence>\n" +
+                "   </job>\n" +
+                "</joblist>"
+
+        def nowDate = LocalDateTime.now(ZoneId.of("GMT"))
+        def upDate = nowDate.plusSeconds(10)
+
+        def ny = upDate.year
+        def nmo = upDate.monthValue
+        def nd = upDate.dayOfMonth
+        def nh = upDate.hour
+        def nm = upDate.minute
+        def ns = upDate.second
+
+        def xmlJobAux = xmlJob.replaceAll('xml-args', 'echo hello there')
+                .replaceAll('xml-project-name', PROJECT_NAME)
+                .replaceAll('P_NY', ny.toString())
+                .replaceAll('P_NMO', String.format('%02d', nmo))
+                .replaceAll('P_ND', String.format('%02d', nd))
+                .replaceAll('P_NH', String.format('%02d', nh))
+                .replaceAll('P_NM', String.format('%02d', nm))
+                .replaceAll('P_NS', String.format('%02d', ns))
+        def tempFile = File.createTempFile("temp", ".xml")
+        tempFile.text = xmlJobAux
+        tempFile.deleteOnExit()
+        tempFile.path
+    }
+
+    def generatePath1() {
```

Comment:
this should go in "JobUtils.groovy" and have a proper name and gdoc

there is no point on having the xml definition and a replace statement after